### PR TITLE
feat: clean build cache when PR closed

### PR DESCRIPTION
**Pull request by gythialy**
- cleanup caches by a branch
- fix welcome workflow

Signed-off-by: gythialy <gythialy.koo+github@gmail.com>

### CODE REVIEW

**Review by developer-guy — APPROVED** [review]

File: .github/workflows/welcome.yml [hunk 1]
```diff
@@ -1,28 +1,18 @@
...
-            Thanks for opening this pull request! Please check out our contributing guidelines.
-
-          FIRST_PR_MERGED: |
-            🎉 @{{ author }}
-            Congrats on merging your first pull request! We here at behaviorbot are proud of you!
-
-          STAR_MESSAGE: If you like this project, please ⭐star⭐ our repo.
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          issue-message: "Hey, thanks for opening your first issue! 🙂 Be sure to check out the contribution guidelines if you haven't already. Thanks!"
+          pr-message: "Welcome, new contributor! congrats on your first PR! 👋👋👋"
```
Comment (cpanato) [inline .github/workflows/welcome.yml:18]: Nit: new line

File: .github/workflows/clean-cache.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+          ## Setting this to not fail the workflow while deleting cache keys. 
+          set +e
+          echo "Deleting caches..."
+          for cacheKey in $cacheKeysForPR
+          do
+              gh actions-cache delete $cacheKey -R $REPO -B $BRANCH --confirm
+          done
+          echo "Done"
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (cpanato) [inline .github/workflows/clean-cache.yaml:34]: Nit: new line

Comment (cpanato) [inline .github/workflows/clean-cache.yaml:34]: You might need to setup a permission in this job. I think it will require the actions permission

**Review by cpanato — COMMENTED** [review]
Few comments

File: .github/workflows/welcome.yml (hunk 1 above)
Comment (gythialy) [reply .github/workflows/welcome.yml]: doen.

File: .github/workflows/clean-cache.yaml (hunk 2 above)
Comment (gythialy) [reply .github/workflows/clean-cache.yaml]: add 
```yaml 
permissions:
  actions: write
```

Comment (gythialy) [reply .github/workflows/clean-cache.yaml]: done.

File: .github/workflows/welcome.yml (hunk 1 above)
Comment (cpanato) [reply .github/workflows/welcome.yml]: cool thanks

**Review by cpanato — APPROVED** [review]

**Pull request merged**